src/components/Kv/KvLightbox.vue

```diff
@@ -105,6 +112,11 @@ export default {
 	},
 	mounted() {
 		this.isShown = this.visible;
+
+		if (this.$slots.title) {
+			this.$slots.title[0].elm.id = 'title';
```

Comment:
vendors~app.js:90792 [Vue warn]: Error in mounted hook: "TypeError: Cannot set property 'id' of undefined"

Reply:
Good catch. Removing this block completely still causes the focus lock to escape (though no console error). There's something more going on I think.

Reply:
I believe this console error has to do with some deprecated slot syntax we are using. It points to src/pages/Autolending/SaveButton.vue which is using this slot syntax: `#title` Which has been deprecated in 2.6 https://vuejs.org/v2/guide/components-slots.html#Named-Slots-with-the-slot-Attribute

We should make a new item to refactor all slot use to be consistant with the new syntax.

Reply:
I believe the best approach here is to make sure you are setting the id="title" in the components using kv-lightbox. In kv-lightbox you can check for the existence of the title slot and add the aria-labelled by property. Reference: https://github.com/vuejs/vue/issues/9580 this.$slots.title[0].elm.id is a hack I don't think we should use it

Reply:
Adding the ID in the parent component would be less hacky, but every time someone makes a lightbox they'd have to know to add the ID of "title" to what they put in the header slot, so it probably won't happen much. Then the aria-labelledby could be wrong which is probably worse than not being there at all.

A different approach could be to add a prop to kv-lightbox called "title" which takes a string. If the prop is there we always render it inside an h2 with the id, and can then attach the aria-labelledby. An upside is more consistent looking dialogs, a downside is less visual flexibility, though I suppose you could ::v-deep the styling if necessary. And less flexibility to put things beyond strings in the title. I think there's only one instance of that in [AutolendingWho.vue](https://github.com/kiva/ui/blob/2dbcbcc8a2fd6f2b3b0b3d9e44da0c62325dd869/src/pages/Autolending/AutolendingWho.vue#L26)

Reply:
I'm starting to like the prop idea more. If you want something custom you not use the title prop and build a special title in the body, but the majority of the dialogs would be consistent and handle the labelled-by out of the gate. @mcstover @emuvente thoughts?

Reply:
Yeah I think a prop is a good way to go about it, especially since, as you point out, ::v-deep can be used if alternative styling is necessary.

Reply:
I think the prop approach is fine and then for other cases where we need to have a more fully featured header we can just add and style it through the main slot...

Reply:
Cool, removed the arial-labelledby block for this PR. Will open a different with the prop change